src/content/docs/release-notes/agent-release-notes/python-release-notes/python-agent-80500.mdx

```diff
@@ -15,7 +15,7 @@ Install the agent using `easy_install/pip/distribute` via the [Python Package In
 
 * **Add support for redis v4.3.5 & v4.4.0 methods**
```

Comment:
@hmstepanek How about changing the ampersand to 'and' on line 16, too?

Reply:
Fixed